plugins/ilias/ilias_

```diff
@@ -0,0 +1,254 @@
+        print("graph_info Number of active ILIAS user sessions")
+        print("graph_vlabel ilsessions")
+        print("graph_category cms")
+        print("ilsessions.label ilSessions")
+        print("ilsessions.min 0")
+        print("ilsessions.draw AREA")
+
+    def execute_sessions(self):
+        cursor = self.con.cursor()
+        cursor.execute(
+            "SELECT COUNT( user_id ) "
+            "FROM usr_session "
+            "WHERE `expires` > UNIX_TIMESTAMP( NOW( ) ) AND user_id != 0"
+        )
+        usrs = cursor.fetchone()[0]
+        print("ilsessions.value %s" % (usrs))
+
+    def config_5minavg(self):
+        print("graph_title ILIAS 5 avg")
+        print("graph_info ILIAS sessions created or "
+              "updated during the last 5 minutes")
+        print("graph_vlabel il5minavg")
+        print("graph_category cms")
+        print("il5minavg.label 5 min Count")
+        print("il5minavg.min 0")
+        print("il5minavg.draw AREA")
+
+    def execute_5minavg(self):
+        cursor = self.con.cursor()
+        cursor.execute(
+            "SELECT COUNT( user_id ) "
+            "FROM usr_session "
+            "WHERE 5 * 60 > UNIX_TIMESTAMP( NOW( ) ) - ctime AND user_id != 0"
+        )
+        usrs = cursor.fetchone()[0]
+        print("il5minavg.value %s" % (usrs))
+
+    def config_60minavg(self):
+        print("graph_title ILIAS 60 avg")
+        print("graph_info ILIAS sessions created or "
+              "updated during the last 60 minutes")
+        print("graph_vlabel il60minavg")
+        print("graph_category cms")
+        print("il60minavg.label 60 min Count")
+        print("il60minavg.min 0")
+        print("il60minavg.draw AREA")
+
+    def execute_60minavg(self):
+        cursor = self.con.cursor()
+        cursor.execute(
+            "SELECT COUNT( user_id ) "
+            "FROM usr_session "
+            "WHERE 60 * 60 > UNIX_TIMESTAMP( NOW( ) ) - ctime AND user_id != 0"
+        )
+        usrs = cursor.fetchone()[0]
+        print("il60minavg.value %s" % (usrs))
+
+    def config_total1day(self):
+        print("graph_title Users in 24h")
+        print("graph_info ILIAS users logging in during last 24h")
+        print("graph_vlabel iltotal1day")
+        print("graph_category cms")
+        print("iltotal1day.label User/24h")
+        print("iltotal1day.min 0")
+        print("iltotal1day.draw AREA")
+
+    def execute_total1day(self):
+        cursor = self.con.cursor()
+        cursor.execute(
+            "SELECT COUNT( usr_id ) "
+            "FROM `usr_data` "
+            "WHERE last_login >= DATE_SUB( NOW( ) , INTERVAL 1 DAY )")
+        usrs = cursor.fetchone()[0]
+        print("iltotal1day.value %s" % (usrs))
+
+    def run(self):
+        cmd = ((len(sys.argv) > 1) and sys.argv[1] or None) or "execute"
+        function = None
+
+        if cmd == "config":
+            function = "config"
+        elif cmd == "suggest":
+            print ("sessions")
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) does not like whitespace between function and arguments.
May try flake8 or something similar for a style check?
(sadly this is not part of our automatic testing environment, yet)

Regarding pep8online: it does not seem to check the above issue. I reported this to the author of that service.